src/adapters/EOFeedAdapter.sol

```diff
@@ -37,24 +39,31 @@ contract EOFeedAdapter is IEOFeedAdapter, Initializable {
      * @notice Initialize the contract
      * @param feedManager The feed manager address
      * @param feedId Feed id
-     * @param feedDecimals The decimals of the rate
+     * @param inputDecimals The input decimal precision of the rate
+     * @param outputDecimals The output decimal precision of the rate
      * @param feedDescription The description of feed
      * @param feedVersion The version of feed
      */
     function initialize(
         address feedManager,
         uint16 feedId,
-        uint8 feedDecimals,
+        uint8 inputDecimals,
+        uint8 outputDecimals,
         string memory feedDescription,
         uint256 feedVersion
     )
         external
         initializer
     {
         if (feedManager == address(0)) revert InvalidAddress();
+        if (inputDecimals == 0 || inputDecimals > 18) revert InvalidDecimals();
```

Comment:
Why we don't allow 0 decimals? I believe there are tokens with 0 decimality

Reply:
Correct, will remove the input checks - same for '>18'